inbox/react/get-started.mdx

```diff
@@ -4,73 +4,104 @@ sidebarTitle: "Get Started"
 description: "Learn how to add novu powered In-App notification center to your React app"
 ---
 
-Novu provides the `@novu/notification-center` a React library that helps to add a fully functioning notification center to your web application in minutes. Let's do a quick recap on how you can easily use it in your application.
+Novu provides the `@novu/react` a React library that helps to add a fully functioning notification center to your web application in minutes. Let's do a quick recap on how you can easily use it in your application.
```

Comment:
✏ **suggestion:** I think that we should leave the old NC documentation pages in the old version of docs, but the new Inbox pages should be listed as the main component now.

Reply:
Correct, it should be part of V1 docs